core/src/entities/formatHandler.js

```
@@ -1,5 +1,32 @@
+    }
+    if (normalizedSeg.notes !== null && typeof normalizedSeg.notes === 'object') { // unfortunately null is an object
+        // populate ph samples from comments
+        if (normalizedSeg.notes.ph) {
+            for (const part of normalizedSeg.nstr) {
+                if (part.t === 'x' && normalizedSeg.notes.ph[part.v]?.sample !== undefined && part.s === undefined) {
+                    part.s = normalizedSeg.notes.ph[part.v].sample;
+                }
+            }
+        }
+    } else {
+        delete normalizedSeg.notes;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)